app/web/package.json

```diff
@@ -34,6 +34,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "cross-env CI=true react-scripts build",
+    "postbuild": "react-snap",
```

Comment:
Do we still want to leave this as a postbuild step?

Reply:
Yes since 200.html is the original index.html